README.md

```diff
@@ -173,6 +183,73 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `QueryPallet`
+
+Query the existence of a particular pallet type.
+
+- `name: Vec<u8>`: The name of the pallet to query.
+- `query_id`: The value to make the returned message identifiable with this query.
```

Comment:
what will be the query_id type? Compact u32? Or just bytes so we can use hash of the name to avoid the need to store id => name mapping?

Reply:
we can probably just do `u64` and hash the bytes.